lib/Translation/ExportDataExtractorService/DataExtractor/DocumentDataExtractor.php

```diff
@@ -71,21 +71,21 @@ public function extract(TranslationItem $translationItem, string $sourceLanguage
      */
     protected function addDoumentTags(Document $document, AttributeSet $result): DocumentDataExtractor
     {
-        $elements = [];
+        $editables = [];
         $service = new Document\Service;
 
         $translations = $service->getTranslations($document);
 
         if ($document instanceof Document\PageSnippet) {
             $tagNames = $this->tagUsageResolver->getUsedTagnames($document);
```

Comment:
We should use `$editableNames` instead of `$tagNames` and also `$editable` instead of `$tag` ...

Reply:
$tag related changes should be a part of https://github.com/pimcore/pimcore/issues/6888 ..right?